src/qt/receivecoinsdialog.cpp

```diff
@@ -267,9 +267,18 @@ void ReceiveCoinsDialog::copyColumnToClipboard(int column)
 // context menu
 void ReceiveCoinsDialog::showMenu(const QPoint &point)
 {
-    if (!selectedRow().isValid()) {
+    const QModelIndex sel = selectedRow();
+    if (!sel.isValid()) {
         return;
     }
+
+    // disable context menu actions when appropriate
+    const RecentRequestsTableModel * const submodel = model->getRecentRequestsTableModel();
+    const RecentRequestEntry * const req = &submodel->entry(sel.row());
```

Comment:
wouldn't a const reference make more sense?
```suggestion
    const RecentRequestEntry& req = submodel->entry(sel.row());
```

Reply:
addressed in 3f83f05f52be8f3bac2267d447b5037409cde17d